homeassistant/components/plex/server.py

```diff
@@ -186,8 +186,12 @@ def refresh_entity(self, machine_identifier, device, session):
             session,
         )
```

Comment:
It's hard to see in this diff, but we need to change to use the async version `async_dispatcher_send` in `refresh_entity` since that's called from async context.

Reply:
Addressed in #33785.